Shared/Samples/Render multilayer symbols/RenderMultilayerSymbolsView.swift

```diff
@@ -0,0 +1,453 @@
+    /// - Returns: An `Array` of new `Graphic` objects to display on the map.
+    private func makeMultilayerPolygonGraphics() -> [Graphic] {
+        // Create a text graphic.
+        var graphics = [Graphic(
+            geometry: Point(x: 65, y: 50, spatialReference: .wgs84),
+            symbol: makeTextSymbol(text: "Multilayer\nPolygons")
+        )]
+        
+        // Create multilayer polygon symbols.
+        graphics.append(contentsOf: [
+            // Cross-hatched diagonal lines.
+            makeMultilayerPolygonSymbolGraphic(angles: [-45, 45], offset: 0),
+            // Hatched diagonal lines.
+            makeMultilayerPolygonSymbolGraphic(angles: [-45], offset: offset),
+            // Hatched vertical lines.
+            makeMultilayerPolygonSymbolGraphic(angles: [90], offset: offset * 2)
+        ])
+        return graphics
+    }
+    
+    /// Creates a graphic of a multilayer polygon symbol.
+    /// - Parameters:
+    ///   - angles: The `Double`angles at which to draw the fill lines within the polygon.
+    ///   - offset: The `Double` used to keep a consistent distance between symbols in the same column.
+    /// - Returns: A new `Graphic` object of the multilayer polygon symbol.
+    private func makeMultilayerPolygonSymbolGraphic(angles: [Double], offset: Double) -> Graphic {
+        // Create a stroke symbol layer to make the symbol layer.
+        let hatchStrokeLayer = SolidStrokeSymbolLayer(width: 2, color: .red)
+        
+        // Create a list to hold all necessary symbol layers.
+        var symbolLayers: [SymbolLayer] = []
+        
+        // For each angle, create a hatch fill symbol layer with hatched lines at the given angle.
+        for angle in angles {
+            let hatchLayer = HatchFillSymbolLayer(
+                polylineSymbol: MultilayerPolylineSymbol(symbolLayers: [hatchStrokeLayer]),
+                angle: angle
+            )
+            
+            // Set separation distance for lines.
+            hatchLayer.separation = 9
+            symbolLayers.append(hatchLayer)
+        }
+        
+        // Create a stroke symbol layer to be used as an outline.
+        let outlineStrokeLayer = SolidStrokeSymbolLayer(width: 1, color: .black)
+        symbolLayers.append(outlineStrokeLayer)
+        
+        // Create a multilayer polygon symbol from the symbol layer list.
+        let polygonSymbol = MultilayerPolygonSymbol(symbolLayers: symbolLayers)
+        
+        // Create rectangle polygon for the graphic.
+        let polygonBuilder = PolygonBuilder(spatialReference: .wgs84)
+        polygonBuilder.add(Point(x: 60, y: 25 - offset))
+        polygonBuilder.add(Point(x: 70, y: 25 - offset))
+        polygonBuilder.add(Point(x: 70, y: 20 - offset))
+        polygonBuilder.add(Point(x: 60, y: 20 - offset))
+        
+        // Create a graphic of the multilayer polygon symbol.
+        return Graphic(geometry: polygonBuilder.toGeometry(), symbol: polygonSymbol)
```

Comment:
Why do we need to use a builder here?

We could just create the polygon using: [Polygon(points:)](https://developers.arcgis.com/swift/api-reference/documentation/arcgis/multipart/init(points:))